blocks/model.py

```diff
@@ -156,7 +157,11 @@ def __init__(self, outputs):
             if brick not in children and brick not in self.top_bricks:
                 self.top_bricks.append(brick)
         if len(set(b.name for b in self.top_bricks)) < len(self.top_bricks):
-            raise ValueError("top bricks with the same name")
+            # find the first occurences and return it in the error message
+            liste_name = [b.name for b in self.top_bricks]
+            occ = liste_name[numpy.argmax([liste_name.count(liste_name[i])
+                            for i in xrange(len(liste_name))])]
```

Comment:
It's pretty redundant to call it `list_of...` of `..._list` actually. Just `names` will do.